packages/gdl-frontend/locale/en/en.json

```diff
@@ -0,0 +1,118 @@
+{
+  "Blog": "Blog",
+  "From": "From",
+  "Home": "Home",
+  "More": "More",
+  "Next": "Next",
+  "Sync": "Sync",
+  "from": "from",
+  "About": "About",
+  "Done!": "Done!",
+  "Level": "Level {level}",
+  "Share": "Share",
+  "Log in": "Log in",
+  "Oh no!": "Oh no!",
+  "Search": "Search",
+  "result": "{totalCount, number} {totalCount, plural, one {result for} other {results for}}",
+  "Level 1": "Level 1",
+  "Level 2": "Level 2",
+  "Level 3": "Level 3",
+  "Level 4": "Level 4",
+  "License": "License",
+  "Log out": "Log out",
+  "Similar": "Similar",
+  "Tooltip": "Tooltip",
+  "Copy URL": "Copy URL",
+  "Download": "Download",
+  "Favorite": "Favorite",
+  "Featured": "Featured",
+  "Next tip": "Next tip",
+  "Previous": "Previous",
+  "Selected": "Selected:",
+  "Welcome!": "Welcome!",
+  "Decodable": "Decodable",
+  "Favorites": "Favorites",
+  "GDL Admin": "GDL Admin",
+  "Open menu": "Open menu",
+  "Read book": "Read book",
+  "Translate": "Translate",
+  "Categories": "Categories",
+  "Close book": "Close book",
+  "More books": "More books",
+  "Read aloud": "Read aloud",
+  "Contributor": "{contributorType}",
+  "Contributors": "{contributorType}s",
```

Comment:
Disse to kommer fra no flertall bøyninger. Går det an å sette de statisk og ikke dynamisk? Hvis en oversetter ser dette så tenker jeg de ikke skjønner hvordan de skal oversette denne verdien.

Reply:
Hvis vi skriver om til egne komponenter for `Author`, `Illustrator` osv, så kan vi det. Evt om vi bare har flertall/entallbøyningen av feks `Author` ?